readyapi4j-swagger/src/test/java/com/smartbear/readyapi4j/swagger/SwaggerTestStepBuilderTest.java

```diff
@@ -29,8 +36,37 @@ public void testExistingOperation() throws Exception {
     public void testMissingOperation() throws Exception {
         try {
             petstore.operation("tjoho");
-            assertFalse("Expected IllegalArgumentException for invalid operationId", true);
+            fail("Expected IllegalArgumentException for invalid operationId");
```

Comment:
You can use the @Rule annotation to do that in a more elegant way. See here: http://blog.codeleak.pl/2013/07/3-ways-of-handling-exceptions-in-junit.html